installation.md

```diff
@@ -89,7 +89,7 @@ Docker Compose initializes both a PostgreSQL and a Redis database.
 To connect to these databases, use the following commands:
 
 ```bash
-docker compose exec db psql postgres://username:password@db:5432/dbname
+docker compose exec db psql postgres://moncomptepro:moncomptepro@db:5432/moncomptepro
```

Comment:
What's changed with the database? (it's my curiosity)

Reply:
The doc has not been updated after #685 !